src/azure-cli-core/azure/cli/core/commands/constants.py

```diff
@@ -9,7 +9,7 @@
 
 CLI_COMMON_KWARGS = ['min_api', 'max_api', 'resource_type', 'operation_group',
                      'custom_command_type', 'command_type', 'is_preview', 'preview_info',
-                     'is_experimental', 'experimental_info']
+                     'is_experimental', 'experimental_info', 'local_context_attribute']
```

Comment:
as we synced offline, is it really necessary to add the attribute chaneges in this release? this release suppose be a light change only for resource group by extending existing az config.